nixos/tests/shiori.nix

```diff
@@ -1,80 +1,79 @@
-import ./make-test-python.nix ({ pkgs, lib, ...}:
+import ./make-test-python.nix ({ pkgs, lib, ... }:
```

Comment:
In future please separate formatting commits from commits that changes stuff to speed up the review process.

Reply:
ah sorry! I restored this file and didn't intended to bring it in the commit, this sneaked through me.